### PR TITLE
Added new init action for Datasketches

### PR DESCRIPTION
**Pull request by kuldeepkk-dev**
Added new initialization action for Apache Datasketches.

### CODE REVIEW

**Comment by cjac** [comment]
You'll need to add a BUILD file and some tests for the CI framework for gcbrun to trigger.  Check with Erick ; he recently added some to an action that was missing them.

File: datasketches/datasketches.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#set -x
```
Comment (functicons) [inline datasketches/datasketches.sh:2]: This should not be commented out. And the best practice is to use `set -euxo pipefail`

File: datasketches/datasketches.sh [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+declare -A all_components=( [java]="3.1.0" [hive]="1.2.0" [memory]="2.0.0" [pig]="1.1.0" )
+
+#Download and configure datasketches libraries
+for lib in "${!all_components[@]}" 
+do
+	configure_libraries $lib ${all_components[$lib]}
+done
+
+#Deploy spark java thetasketches example jar
```
Comment (functicons) [inline datasketches/datasketches.sh:62]: Prefer extracting helper functions over comments. Consider structure like:

function main() {
  init
  configure_libraries
  download_example_jar
}

**Review by functicons — CHANGES_REQUESTED** [review]

**Review by functicons — APPROVED** [review]
LGTM

**Pull request merged**